stories/Components/TableStoriesDocs/TableDocs.mdx

```diff
@@ -1,42 +1,59 @@
 `import { Table } from "@bigbinary/neetoui";`
 
-We use Ant Design's `Table` component under the hood. For extra customization, refer [AntD Table](https://ant.design/components/table/#API).
+```jsx
+const columnData = [
+  {
+    title: "ID",
+    dataIndex: "id",
+    key: "id",
+    width: 75,
+    sorter: (a, b) => a.id - b.id,
+  },
+  // Other column defenitions
+  ]
+
+const rowData = [
+  {
+    id: "1",
+    // Data for other columns
+  },
+  {
+    id: "2",
+    // Data for other columns
+  }
 
-Please follow the below syntax for `columnData` and `rowData`.
+  // Other rows
 
-```js
-const columns = [
-    {
-      title: "ID",
-      dataIndex: "id",
-      key: "id",
-      width: 75,
-      sorter: true;
-    },
-    {
-      title: "GUID",
-      dataIndex: "guid",
-      key: "guid",
-      width: 150,
-      ellipsis: {
-        showTitle: false,
-      },
-    },
-  ];
+];
 
-  const handleTableChange = (pagination, filters, sorter) => {
-    console.log(pagination, filters, sorter);
-  };
 
+<Table
+  columnData = {columnData}
+  rowData = {rowData}
+  />
```

Comment:
Will we be able to successfully render a neetoUI table by just passing these two props alone? If not, then let's also point out the essential props.